runtime/permissions/lib.rs

```diff
@@ -720,13 +721,14 @@ fn format_display_name(display_name: Cow<'_, str>) -> Cow<'_, str> {
 #[derive(Debug, Clone, Eq, PartialEq)]
 enum AllowOrDenyDescRef<'a, TAllowDesc: AllowDescriptor> {
   Allow(&'a TAllowDesc),
-  Deny(&'a TAllowDesc::DenyDesc, /* is prompt */ bool),
+  Deny(&'a TAllowDesc::DenyDesc, /* precedence */ u8),
```

Comment:
Looking at the code, it looks like precedence of 0 == highest precedence. maybe add a comment to that effect? Or switch to an enum as bartek suggested